app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -619,12 +602,93 @@ class MainActivity : AppCompatActivity(), MainViewController, RouteCallback,
         searchResultsView.onSearchResultsSelectedListener = this
     }
 
+    private fun baseAttributionParams(): RelativeLayout.LayoutParams {
```

Comment:
Wow this so much code to manipulate layout params. I wish there was a better way to do this in Android 😢 

One idea I had was to just adjust the layout params on `MapView` itself now that the find me button and attribution are part of it. But I am guessing that would have the side effect of changing the position of the map if we mess with the layout params on the view?